app/bundles/EmailBundle/Entity/Email.php

```diff
@@ -1070,4 +1076,20 @@ public function getReadPercentage($includevariants = false)
             return 0;
         }
     }
+
+    /**
+     * @return mixed
+     */
+    public function getCustomData()
+    {
+        return $this->customData;
+    }
+
+    /**
+     * @param mixed $customData
+     */
+    public function setCustomData($customData)
+    {
+        $this->customData = $customData;
+    }
```

Comment:
The setters should `return $this` so they could be chained like `$email->setCustomData($customData)->setEmailType($emailType);`